packages/ra-ui-materialui/src/layout/PageTitleConfigurable.tsx

```diff
@@ -34,7 +39,24 @@ export const PageTitleConfigurable = ({ preferenceKey, ...props }) => {
                 },
             }}
         >
-            <PageTitle {...props} />
+            <PageTitleConfigurableInner {...props} />
         </Configurable>
     );
 };
+
+const PageTitleConfigurableInner = ({ className, ...rest }: any) => {
```

Comment:
**suggestion:** We should avoid `any` to type props. 

`className` seems to be not passed to this component (see `<PageTitleConfigurable>`). Yet, `className` is present in the `...rest` props. So you can remove `className` and avoid `any`.

```suggestion
const PageTitleConfigurableInner = ({ ...rest }) => {
```